README.md

```diff
@@ -1,4 +1,4 @@
-# Lombok.NET
+~~~~# Lombok.NET
```

Comment:
@SimonCropp Why this change? The README file looks weird now when displayed on the repository start page.

Reply:
I have no idea. want a pr to revert that bit?

Reply:
No worries, I'll take care of it :)